continuous_integration/gpuci/axis.yaml

```diff
@@ -0,0 +1,13 @@
+PYTHON_VER:
+- "3.8"
+
+CUDA_VER:
+- "11.2"
+
+LINUX_VER:
+- ubuntu18.04
+
+RAPIDS_VER:
+- "21.10"
```

Comment:
should this be `21.12`? I noticed the PR is going into `branch-21.12`

Reply:
It should, but we would first need to build images for Dask + RAPIDS 21.12 by specifying that [here](https://github.com/rapidsai/dask-build-environment/blob/0e85f38b2559707fd25175ab111cd828f8423d4f/ci/axis/dask.yaml#L20-L22). Typically we couple each RAPIDS release with a corresponding UCX-Py version, but I'm not sure if `branch-0.23` of ucx-py has nightlies up yet

Reply:
In that case I think we might be good to merge and update the Rapids requirement post 21.10 release

Reply:
Sounds good to me! Ideally in the future I hope we can put some automation in place so that PRs are automatically opened across all the relevant Dask gpuCI repos on release (might make sense to have this be a gpuCI job but could also be handled by GHA)